packages/common/src/index.ts

```diff
@@ -423,6 +429,16 @@ export type TablesConfiguration = {
     };
 };
 
+export type CreateProjectMember = {
+    email: string;
```

Comment:
could it be `userUuid` rather than email?

Reply:
Not on createProject because we are adding the users with their emails , everything else is using userUUid